CONTRIBUTING.md

```diff
@@ -62,9 +62,24 @@ Here are a few examples:
 * At the end of the entry, add an implicit link to your GitHub user page as `([@username][])`.
 * Alternatively, you may modify the CHANGELOG file directly, but this may result in conflicts later on. Also, if this is your first contribution to RuboCop project, add a link definition for the implicit link to the bottom of the changelog as `[@username]: https://github.com/username`.
 
+### Real World Rails
+
+[Real World Rails][7] is a collection of open source Rails application codebases.
+When introducing or changing a cop, running against these can help to uncover wide range of potential problems.
+
+WIP:
```

Comment:
Suggestion:
------------

Run the following to check out Real World Rails:
```sh
git clone git@github.com:eliotsykes/real-world-rails.git
cd real-world-rails

# This will take some time...
git submodule update --init --remote --checkout --single-branch --depth 1

rm -v **/.rubocop.yml
```

Now, you can run `rubocop --only <your-cop-name> <path-to-real-world-rails>`.

Take a close look at the output and make sure that:
* there are no errors
* the cop doesn't stall

Ideally, the cop should only raise legitimate offenses against Real World Rails.

If there are false positives, mark the cop as unsafe.